src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/Coverage/TouchstoneCoverageTests.kt

```diff
@@ -50,6 +65,114 @@ class TouchstoneCoverageTests : CoverageTests()
         test.run()
     }
 
+    @Test
```

Comment:
So are there also tests for scenarios _without_ subnational rows that check the expected values are returned?